src/ukify/ukify.py

```diff
@@ -59,7 +59,7 @@
     'x86_64'       : ['x64', 'ia32'],
     'i[3456]86'    : ['ia32'],
     'aarch64'      : ['aa64'],
-    'arm[45678]*l' : ['arm'],
+    'armv[45678]*l': ['arm'],
```

Comment:
https://stackoverflow.com/questions/45125516/possible-values-for-uname-m says that possible values are `arm`, `armv8b`, `armv8l`. This doesn't seem very trustworthy ;)    https://en.wikipedia.org/wiki/Uname#Examples has a better list, with `armv7l` and `armv6l`. So this change seems correct.  We could probably also drop the star from the glob, but we can also do that later.